user_tools/src/spark_rapids_tools/storagelib/csppath.py

```diff
@@ -56,6 +57,7 @@
 
 
 CspPathString = Annotated[str, StringConstraints(pattern=r'^\w+://.*')]
+AZURE_STORAGE_PREFIX = 'abfss://'
```

Comment:
We don't need this because this overrides the purpose of the design that each Class will be able to recognize its own URL.

Reply:
removed this, thanks!